example/android/settings.gradle

```diff
@@ -1,15 +1,25 @@
-include ':app'
```

Comment:
Might as well use the `settings.gradle.kts` version, right? Like https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/app/android.tmpl/settings.gradle.kts.tmpl